packages/opencensus-core/test/test-view.ts

```diff
@@ -180,6 +181,59 @@ describe('BaseView', () => {
        });
   });
 
+  describe('getMetric()', () => {
+    const measurementValues = [1.1, 2.3, 3.2, 4.3, 5.2];
+    const buckets = [2, 4, 6];
+    const tags: Tags = {testKey1: 'testValue', testKey2: 'testValue'};
+    const aggregationType = AggregationType.DISTRIBUTION;
```

Comment:
Please add more tests for other AggregationType (COUNT, SUM and LAST_VALUE)

Reply:
Done

Reply:
I don't see tests for AggregationType. COUNT, AggregationType. SUM and AggregationType. LAST_VALUE; Let me know if I miss something.

Reply:
Please ignore this comment, just saw the tests.